src/utils.ts

```diff
@@ -35,6 +38,10 @@ export class Utils {
         return execa(cmdArgs[0], cmdArgs.slice(1), {cwd});
     }
 
+    static syncSpawn (cmdArgs: string[], cwd = process.cwd()): {stdout: string; stderr: string} {
```

Comment:
created a sync version of `Utils.spawn`, because `Utils.getRulesResult` is not async